content/05.main-text.md

```diff
@@ -749,10 +767,68 @@ Producing results for these queries relies on internal calls to the PostgreSQL r
+
+### Visual Design
+
+A limited palette of colors was chosen to represent the different types of nodes (metanodes) in the Hetionet knowledge graph.
+These colors are listed and programmatically accessible in the [`hetionet`](https://github.com/hetio/hetionet) repository under `/describe/styles.json`. 
+
+The palette is used in all Hetio-related applications and materials.
+This is not just to maintain a consistent look and feel across the Hetio organization, but to convey clear and precise meaning.
+For example, the colors used in the metagraph in Figure {@fig:rephetio}A are exactly the same colors, and thus represent the same types of entities, as in _any part of the connectivity search app_ (Figure {@fig:webapp}).
+
+Colors in the palette are also used in the Hetio logo (seen in Figure {@fig:website}) and other miscellaneous logos and iconography across the website, to establish an identifiable "brand" for Hetio organization as a whole.
+
+### Webapp and Frontend
+
+We developed the [connectivity search app](https://het.io/search), as well as the related [rephetio](https://het.io/repurpose/) and [disease-associated genes](https://het.io/disease-genes/) apps, as single-page, standalone applications using React and associated tools.
+The source code for the connectivity search app is available in the [`connectivity-search-frontend`](https://github.com/greenelab/connectivity-search-frontend) repository, and the source code for the related apps are available in the [hetio GitHub organization](https://github.com/hetio).
+
+Since the rest of the overarching Hetio website was mostly non-interactive content, it was appropriate to construct the bulk of the website in simple static formats like Markdown and HTML using Jekyll, and leave React for implementing the sections of the site that required more complex behavior.
+
+We used React's own `create-react-app` command line tool to generate a boilerplate for the app.
+This greatly simplified setting up and maintaining the app's testing and building pipelines, bypassing time-consuming configuration of things like Webpack and linters.
+Some configuration of was necessary to produce non-hashed, consistently named output files like `index.js` that could be easily and reliably referenced by and embedded into the Hetio Jekyll website.
+
+For authoring components, we used React's traditional class syntax.
+At the time of authoring the app, React Hooks were still nascent, thus the simpler and less-verbose functional syntax was not viable.
+
+We created a custom package of React components and utility functions that could be shared across the multiple interactive apps on the website.
+The package consists of interface "components" (building blocks) like buttons, sortable/searchable/paginated tables, etc., as well as utility functions for formatting data, debugging, etc.
+Each of the interactive apps import this package to reduce code repetition, and to enforce a consistent style and behavior across the website.
```

Comment:
Where is this importable package located? Not saying we should link to it in manuscript although maybe in an HTML comment for our reference?

Reply:
I think it makes sense to just have a sentence linking to it. It's fairly essential, and the other key repos are linked to in other sections.